src/Domain/Creators/CylindricalBinaryCompactObject.hpp

```diff
@@ -0,0 +1,297 @@
+ * If \f$c_A\f$ and \f$c_B\f$ are the input parameters center_A and
+ * center_B, \f$r_A\f$ and \f$r_B\f$ are the input parameters radius_A and
+ * radius_B, and \f$R\f$ is the outer boundary radius, we demand the
+ * following restrictions on parameters:
+ * - \f$c_A^0>0\f$; this is a convention to simplify the code.
+ * - \f$c_B^0<0\f$; this is a convention to simplify the code.
+ * - \f$|c_A^0|\le|c_B^0|\f$. We should roughly have \f$r_A c_A^0 + r_B c_B^0\f$
+ *   close to zero; that is, for BBHs (where \f$r_A\f$ is roughly twice the
+ *   mass of the heavier object A, and \f$r_B\f$ is roughly twice the mass
+ *   of the lighter object B) the center of mass should be roughly
+ *   at the origin.
+ * - \f$0 < r_B < r_A\f$
+ * - \f$R \ge 3(|c_A^0|-|c_B^0|)\f$; otherwise the blocks will be too compressed
+ *   near the outer boundary.
+ *
+ */
+class CylindricalBinaryCompactObject : public DomainCreator<3> {
+ public:
+  using maps_list = tmpl::list<
+      domain::CoordinateMap<Frame::Logical, Frame::Inertial,
+                            CoordinateMaps::ProductOf3Maps<
+                                CoordinateMaps::Affine, CoordinateMaps::Affine,
+                                CoordinateMaps::Affine>,
+                            CoordinateMaps::CylindricalEndcap,
+                            CoordinateMaps::DiscreteRotation<3>>,
+      domain::CoordinateMap<
+          Frame::Logical, Frame::Inertial,
+          CoordinateMaps::ProductOf2Maps<CoordinateMaps::Wedge<2>,
+                                         CoordinateMaps::Affine>,
+          CoordinateMaps::CylindricalEndcap,
+          CoordinateMaps::DiscreteRotation<3>>,
+      domain::CoordinateMap<Frame::Logical, Frame::Inertial,
+                            CoordinateMaps::ProductOf3Maps<
+                                CoordinateMaps::Affine, CoordinateMaps::Affine,
+                                CoordinateMaps::Affine>,
+                            CoordinateMaps::CylindricalFlatEndcap,
+                            CoordinateMaps::DiscreteRotation<3>>,
+      domain::CoordinateMap<
+          Frame::Logical, Frame::Inertial,
+          CoordinateMaps::ProductOf2Maps<CoordinateMaps::Wedge<2>,
+                                         CoordinateMaps::Affine>,
+          CoordinateMaps::CylindricalFlatEndcap,
+          CoordinateMaps::DiscreteRotation<3>>,
+      domain::CoordinateMap<
+          Frame::Logical, Frame::Inertial,
+          CoordinateMaps::ProductOf2Maps<CoordinateMaps::Wedge<2>,
+                                         CoordinateMaps::Affine>,
+          CoordinateMaps::CylindricalFlatSide,
+          CoordinateMaps::DiscreteRotation<3>>,
+      domain::CoordinateMap<
+          Frame::Logical, Frame::Inertial,
+          CoordinateMaps::ProductOf2Maps<CoordinateMaps::Wedge<2>,
+                                         CoordinateMaps::Affine>,
+          CoordinateMaps::CylindricalSide,
+          CoordinateMaps::DiscreteRotation<3>>>;
+
+  struct CenterA {
```

Comment:
Are the centers assumed to be on the x-axis? Do we want to impose that restriction?

Reply:
The centers can and will be displaced in the y-direction, as is necessary for the initial data so that the initial ADM linear momentum is zero.  The Domain also accommodates offsets in the z-direction, so we don't need to impose that restriction either (but I think in practice all our runs have z=0 for both objects, so far).

Reply:
ah okay, thanks!